app/i18n/es/gen.php

```diff
@@ -209,6 +209,7 @@
 		'linkedin' => 'LinkedIn',	// IGNORE
 		'mastodon' => 'Mastodon',	// IGNORE
 		'movim' => 'Movim',	// IGNORE
+		'omnivore' => 'Omnivore',	// TODO
```

Comment:
```suggestion
		'omnivore' => 'Omnivore',	// IGNORE
```

And maybe for most languages

Reply:
oh yes, you are right. Thanks for pointing to it

Reply:
fixed